src/steps/fetch-bounties/index.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  createDirectRelationship,
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+  RelationshipClass,
+} from '@jupiterone/integration-sdk-core';
+
+import { IntegrationConfig } from '../../config';
+import { ServicesClient } from '../../client/ServicesClient';
+import { Entities, IntegrationSteps, Relationships } from '../constants';
+import { getAccountEntity, getServiceEntity } from '../../util/jobState';
+import { createBountyEntity, createBountySubmissionEntity } from './converters';
+
+export async function fetchBounties({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const client = new ServicesClient({ apiToken: instance.config.apiToken });
+
+  const accountEntity = await getAccountEntity(jobState);
+  const serviceEntity = await getServiceEntity(jobState);
+
+  await client.iterateBounties(async (bounty) => {
+    const bountyEntity = await jobState.addEntity(createBountyEntity(bounty));
+
+    await jobState.addRelationship(
+      createDirectRelationship({
+        from: accountEntity,
+        to: bountyEntity,
+        _class: RelationshipClass.HAS,
+      }),
+    );
+
+    await jobState.addRelationship(
+      createDirectRelationship({
+        from: serviceEntity,
+        to: bountyEntity,
+        _class: RelationshipClass.MANAGES,
+      }),
+    );
+  });
+}
+
+export async function fetchBountySubmissions({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const client = new ServicesClient({ apiToken: instance.config.apiToken });
+
+  await jobState.iterateEntities(
+    {
+      _type: Entities.BOUNTY._type,
+    },
+    async (bountyEntity) => {
+      const bountyId = bountyEntity.uuid as string | undefined;
+
+      if (!bountyId) {
+        return;
```

Comment:
would be nice to throw a logger.warn here.  `logger` is available in the stepContext which is the first parameter to this method.

Reply:
Will do